material/base.html

```diff
@@ -178,8 +178,8 @@ <h1>{{ page.title | default(config.site_name, true)}}</h1>
       {% endblock %}
     </div>
     {% block scripts %}
-      <script src="{{ 'assets/javascripts/vendor.606193c2.min.js' | url }}"></script>
-      <script src="{{ 'assets/javascripts/bundle.7af7e239.min.js' | url }}"></script>
+      <script src="{{ 'assets/javascripts/vendor.js' | url }}"></script>
```

Comment:
Somehow, the versioning of the vendor bundle got lost. Also from `manifest.json`.

Reply:
Hmm, I just ran `npm run build` and this is what I got. I also get the same on master. I will revert the `npm run build` part for now. It probably is related to Windows. I'll see if I can pinpoint the cause later.